pkgs/tools/security/ghidra/extensions/wasm/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGhidraExtension,
+  ghidra,
+  ant,
+}:
+let
+  version = "2.3.1";
+in
+buildGhidraExtension {
+  pname = "wasm";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "nneonneo";
+    repo = "ghidra-wasm-plugin";
+    rev = "v${version}";
+    hash = "sha256-aoSMNzv+TgydiXM4CbvAyu/YsxmdZPvpkZkYEE3C+V4=";
+  };
+
+  # compile sleigh
+  configurePhase = ''
+    runHook preConfigure
+
+    pushd data
```

Comment:
we should use sourceRoot is possible

Reply:
We're changing directory only for the configure phase (sleigh compilation). sourceRoot is still correct from unpack phase.